data/json/items/gun/ups.json

```diff
@@ -40,8 +40,8 @@
     "id": "gatling_mech_laser",
     "type": "GUN",
     "reload_noise_volume": 20,
-    "name": "CMES laser cannon",
-    "description": "This is the integral weapon system for the CMES exoskeleton mech-suit, a rotating-barrel active-cooled rapid-fire laser system, can spray death downrange with ease.",
+    "name": "laser cannon",
```

Comment:
Might be good to keep such designations for fictional guns too, if someone ever decides to add some variety to them.